adserver/admin.py

```diff
@@ -476,7 +479,7 @@ class AdBaseAdmin(RemoveDeleteMixin, admin.ModelAdmin):
     )
     list_display = readonly_fields[:-3]
     list_select_related = ("advertisement", "publisher")
-    list_filter = ("is_mobile",)
+    list_filter = ("is_mobile", "publisher")
```

Comment:
I just realized this will not cascade down to clicks because `list_filter` is overridden there. We should either add it there or remove the override. I say we just remove the override.

Reply:
I'll add it there and we can remove the override.